infrastructure/iac/iac.go

```diff
@@ -378,7 +378,18 @@ func (iac *Scanner) toIssue(affectedFilePath string, issue iacIssue, fileContent
 		IssueType:           snyk.InfrastructureIssue,
 		CodeActions:         []snyk.CodeAction{action},
 		AdditionalData:      additionalData,
-	}, nil
+	}
+
+	htmlRender, err := NewIacHtmlRender(iac.c)
```

Comment:
nitpick: should this maybe be `renderer` instead of `render`?

Reply:
performance: this would parse the template for each issue (instead of parsing it once and then executing it for each issue). Maybe just add a renderer to the scanner struct and initialize / create it on scanner creation?